app/controllers/api/v1/authentication_controller.rb

```diff
@@ -0,0 +1,21 @@
+class Api::V1::AuthenticationController < ApplicationController
+  before_action :authorize_request, except: :login
+
+  def login
```

Comment:
It's not RESTful name of action
Please, rename it
```suggestion
  def create
```